cnxarchive/utils/ident_hash.py

```diff
@@ -29,6 +29,8 @@ class IdentHashError(Exception):
 
 class IdentHashSyntaxError(IdentHashError):
     """Raised when the ident-hash syntax is incorrect."""
+    def __init__(self, ident_hash):
+        self.ident_hash = ident_hash
```

Comment:
Hrm... I was just looking at the other classes that subclass ``IdentHashError``. They suffer the same issue. Can you fix those too? :pray: